controllers/suite_test.go

```diff
@@ -83,7 +83,7 @@ type customClient struct {
 	onDeleteError error
 }
 
-func (c *customClient) Delete(ctx context.Context, obj client.Object, opts ...client.DeleteOption) error {
+func (c *customClient) Delete(ctx context.Context, obj client.Object, opts ...client.DeleteOption) (err error) {
```

Comment:
leftover from something? (not an issue, just wondering)

Reply:
yes, leftover :+1: